lib/consumer.js

```diff
@@ -55,7 +53,7 @@ Consumer.prototype.getRegistrar = function() {
 Consumer.prototype.getRegistrant = function(address) {
   var self = this;
   return self.registrar.registrantIndexAsync(address, {from: self.address}).then(function(index) { 
```

Comment:
Promisify provide them when applied to a class or object: https://github.com/chronicled/open-registry-sdk/blob/master/lib/provider.js#L226